proptest/CHANGELOG.md

```diff
@@ -23,6 +23,10 @@
 - Add `Arbitrary` impls for `core::num::NonZero*`
 - Adds ability to disable failure persistence via env var `PROPTEST_DISABLE_FAILURE_PERSISTENCE`
 
+### Other Notes
+
+- `proptest` no longer depends on the `quick-error` crate.
```

Comment:
super minor nit, could this say "directly depends" - I think it's worth making the distinction here, since it *does* still depend on it (via rusty-fork)